extensions/wikia/Wall/WallThread.class.php

```diff
@@ -10,8 +10,14 @@ class WallThread {
 	// cached data
 	private $data;
 
-	public function __construct() {
+	private $main_message = false;
+
+	/**
+	 * @param int $id thread ID
+	 */
+	private function __construct( int $id ) {
 		$this->initializeReplyData();
+		$this->mThreadId = $id;
 		$this->mCached = null;
 		$this->mCityId = F::app()->wg->CityId;
```

Comment:
Let's get rid of this field, it's only used in `WallHistory` constructor call which is not needed since table is no longer shared